rdrf/rdrf/static/css/rdrf.css

```diff
@@ -136,6 +136,18 @@ form .aligned ul.errorlist {
   margin-top: 5px;
 }
 
+/**************************************************************************
```

Comment:
Is there somewhere more appropriate to apply styling to the CDEs? Ideally, I'd prefer a react-style approach, but that wouldn't work here.

Reply:
Unfortunately, we're not in the webpack / react word yet, so until then this is the way to do it :-)